book/src/installation_windows.md

```diff
@@ -29,15 +29,17 @@ When choosing the GTK version to build, select `gtk4` instead of `gtk3`:
 ```
 gvsbuild build gtk4
 ```
+<!--  -->
 ### Update `Path` environment variable
 
-2. Update your `Path` environment variable to include the GTK 4 libraries:
+2. Add `New User Variable` in environment variable to include `PKG_CONFIG_PATH`:
    1. Go to settings -> Search and open `Advanced system settings` -> Click on `Environment variables`
-   2. Select `Path` -> Click on `Edit` -> Add `C:\gtk-build\gtk\x64\release\bin`
+   3. Select `New` -> Input `Variable name` : `PKG_CONFIG_PATH` & `Variable value` : `C:\gtk-build\gtk\x64\release\lib\pkgconfig`
```

Comment:
Hah, I guess that was ambiguous.
I didn't mean that you should change the order.
But your instructions are 1., 3. not 1, 2. like they should be.